pyproject.toml

```diff
@@ -66,6 +66,7 @@ pyjwt = "^1.7.1"
 python-json-logger = "^0.1.11"
 pytimeparse = "^1.1.8"
 django-redis = "^4.12.1"
+Adyen = "^3.0.0"
```

Comment:
just wondering about this `^` here. We're pointing to exact versions in requirements and here we have `or higher`

Reply:
Based on that poetry generates requirements with the highest possible version.